network/serde/jsonrpc.go

```diff
@@ -43,8 +44,13 @@ type JsonRpcMessage struct {
 	Message string        `json:"message"`
 	Data    interface{}   `json:"data"`
 }
+
+type PaymentRequest struct {
```

Comment:
This is used to pass the payment information over the RPC interface. Maybe this should be a [voucher](https://github.com/statechannels/go-nitro/blob/d459aba3a5f448a8bb3a97bb81be3c3053a2e193/payments/vouchers.go#L25)?